res/values/strings.xml

```diff
@@ -690,6 +726,14 @@
     <string name="a11y_delivery_status_invalid">Invalid delivery status</string>
     <string name="a11y_message_context_menu_btn_label">Message actions</string>
     <string name="a11y_background_preview_label">Background preview</string>
+    <string name="a11y_profile_image_label">Your profile image</string>
+    <string name="a11y_profile_image_select">set a new profile image</string>
+    <string name="a11y_profile_image_remove">remove your profile image</string>
+    <string name="a11y_qr_data">QR code data</string>
+    <string name="a11y_copy_qr_data">Copied QR url to clipboard</string>
+    <string name="a11y_change_group_image">change group image</string>
```

Comment:
Uppercase first letter?

Reply:
i would just use select_group_image_desktop here. i think this is just a description that would be a button label otherwise.

Reply:
its used for the button that opens the menu where you can set or [set or delete] the group image.

Reply:
i see.